eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -0,0 +1,469 @@
+$VERSION_REGEX = "(?<major>\d+)(\.(?<minor>\d+))?(\.(?<patch>\d+))?((?<pre>[^0-9][^\s]+))?"
+$SDIST_PACKAGE_REGEX = "^(?<package>.*)\-(?<versionstring>$VERSION_REGEX$)"
+
+# Posts a github release for each item of the pkgList variable. SilentlyContinue
+function CreateReleases($pkgList, $releaseApiUrl, $releaseSha) {
+  foreach ($pkgInfo in $pkgList) {
+    Write-Host "Creating release $($pkgInfo.Tag)"
+
+    $releaseNotes = ""
+    if ($pkgInfo.ReleaseNotes[$pkgInfo.PackageVersion].ReleaseContent -ne $null) {
+      $releaseNotes = $pkgInfo.ReleaseNotes[$pkgInfo.PackageVersion].ReleaseContent
+    }
+
+    $isPrerelease = $False
+    if ($pkgInfo.PackageVersion -match $VERSION_REGEX) {
```

Comment:
Doesn't have to be part of this change but at some point we should switch to using our common versioning helpers at https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/SemVer.ps1

Reply:
You're right. I wasn't attempting to introduce new bugs with this PR. 

That is why in the new script I absolutely use `SemVer.ps1`

Reply:
Eh. I'm just going to pull in SemVer usage this PR.